docs/rose-suite-run-proposal/cylc-flow-rc.md

```diff
@@ -0,0 +1,176 @@
+that file contents are exactly the same as they would be in a `suite.rc`.
+Users of the `global.rc` are usually admins or power users and thus changes
+to the format of this file are preferable.
+
+It should be noted that although it will be possible to modify all settings in
+all contexts, that some settings are more likely to be used in global contexts
+and some are more likely to be used in suite configs.
+
+****
+
+## specification for `cylc-flow.rc`
+
+### Top level sections to copy from `suite.rc` to `cylc-flow.rc`
+
+Most sites will leave these to users (Although I could imagine adding a copy-
+right message to meta by default, for example, and one user has suggested a
+very simple runtime might be added too, for training and debugging purposes.)
+
+```ini
+[meta]
+  ...
+[scheduling]
+  ...
+[vizualization]
+  ...
+```
+
+### Top level sections to copy from `global.rc` to `cylc-flow.rc`
+
+It is likely that most users will continue to have these set by site admins.
+
+```ini
+[task events]
+  ...
+[test battery]
+  ...
+[suite host self-identification]
+  ...
+[authentication]
+  ...
+```
+The `[suite servers]` to be deprecated in favour of suite platforms.
+
+
+### Entirely new top level sections
+
+#### `[job platforms]`
+Many of the options in this section will be very similar to `[hosts]`
+It is expected that these will mainly be set in site `cylc-flow.rc`, but that
+small numbers of power users may wish to over-ride them.
+
+```ini
+[job platforms]
+  [[example platform]]
+    run directory =
+    work directory =
+    task communication method =
+    submission polling intervals =
+    execution polling intervals =
+    scp command =
+    ssh command =
+    use login shell =
+    login hosts =                         # list of possible login hosts
+    batch system =                        # name of batch system
+    cylc executable =
+    global init-script =
+    copyable environment variables =
+    retrieve job logs =
+    retrieve job logs command =
+    retrieve job logs max size =      [[default directives]]
+
+    retrieve job logs retry delays =
+    task event handler retry delays =
+    tail command template =
+    [[batch systems]]
+    # Does this need to be like this? Are there really likely to be more than
+    # One batch system per platform.
+      [[__MANY__]]
+        err tailer =
+        out tailer =
+        err viewer =
+        out viewer =
+        job name length maximum =
+        execution time limit polling intervals =
+
+
+    [[default directives]]                # This is probably something to do
+      --some-directive="directive here!"  # sometime after cylc8
+```
+
+#### `[suite platforms]`
+This is the dictionary key formerly known as ``[suite servers]``. Changed only
+for the purpose of keeping the name "platforms" conistent. This is expected to
+be set only by system administrators.
+
+```ini
+[suite platforms]
+```
+
+
+### Top level sections to merge in a more complex way
+#### `[cylc]`
```

Comment:
`[cylc]` should probably become `[general]` or something like that.

Reply:
Yeah, `[cylc]` as a config section heading has annoyed the hell out of me for ages. (My fault too, back in time...)

Reply:
What about `[cylc system/general/settings]`? Do we want to break back compatibility this radically?